9.4/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -51,49 +51,49 @@ Environment variables and volumes
 The image recognizes the following environment variables that you can set during
 initialization by passing `-e VAR=VALUE` to the Docker run command.
 
-**`POSTGRESQL_USER`**  
```

Comment:
These trailing spaces are actually left there on purpose.

See: https://github.com/sclorg/postgresql-container/pull/185#issuecomment-329566358

Reply:
Not only that, white-space changes just cause rush in pull-requests.  Please consider fixing your editor to not remove trailing white-spaces automatically.